src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form--big"
+      data-qa="big"
+    >
+      <label class="search-form__label search-form__label--big">
+        <input
+          data-qa="keypress"
+          class="search-form__input search-form__input--big"
+          type="text"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
```

Comment:
Add empty lines between multiline sibling blocks of HTML. An empty line should be added between the two <form> elements to improve readability.